activation/handler.go

```diff
@@ -371,7 +367,8 @@ func (h *Handler) StoreAtx(ctx context.Context, atx *types.VerifiedActivationTx)
 				}
 				h.log.With().Warning("smesher produced more than one atx in the same epoch",
 					log.Stringer("smesher", atx.NodeID()),
-					log.Inline(atx),
+					log.Object("prev", prev),
+					log.Object("curr", atx),
```

Comment:
Would logging the ID be sufficient?

Reply:
thanks. the same argument can be made about ballots and i have considered that too.
at this stage i prefer to log the whole thing for two reasons:
- there may be bugs in the code that cause honest miners to generate multiple atxs/ballots in an epoch/layer. the extra information will be helpful for us
- if everything works as expected, this occasion should be rare and the extra log won't matter.
i'm open to reduce the log once we have more confidence with the system.